src/styles.scss

```
@@ -0,0 +1,107 @@
+/* You can add global styles to this file, and also import other style files */
+$color-dark: #212529;
+$app-background-color: #d3d3d3;
+$color-white: #ffffff;
```

import and reuse variables
```suggestion
@import ...
```